virtcontainers/store/manager_test.go

```diff
@@ -111,31 +114,36 @@ func TestManagerFindStore(t *testing.T) {
 // TestMain is the common main function used by ALL the test functions
 // for the store.
 func TestMain(m *testing.M) {
-	testDir, err := ioutil.TempDir("", "store-tmp-")
+	setup()
+	rt := m.Run()
+	shutdown()
+	os.Exit(rt)
```

Comment:
OOI, in case you have never seen this - we check for no uses of `os.Exit()` in the main codebase in the CI:
https://github.com/kata-containers/runtime/blob/master/.ci/go-no-os-exit.sh

(I think that is a feature we've had for a very very long time and is little seen, so just raising here for general awareness).
but, maybe that does not apply to test code.

Reply:
Hi~ @grahamwhaley Thanks for the notification. ;).
I'm afraid that it's not applicable for go unit test code, referring from [Go.Doc](https://golang.org/pkg/testing/#hdr-Main):
```
TestMain runs in the main goroutine and can do whatever setup and teardown 
is necessary around a call to m.Run. 
It should then call os.Exit with the result of m.Run.
```